### DISCUSSION TITLE
How can i access the values in url in NextJs 13 for url with #

### DISCUSSION THREAD

**Discussion by alejandrogsk** [vercel/next.js#51375] [App Router]
I have this url "http://localhost:3000/#this-project", i want the "#this-project" value how can i access that?

**Comment by sanalpanicker** [comment]
https://www.example.com/#xyx → app/page.js
get your URL params (#xyz) in index.js and do your logic.

**Reply by alejandrogsk** [reply]
Thanks, but should i use a hook for that? because I'm usgin the App router, so by "index.js" you mean "page.js"? I want to use the jump links from the <a> tag with a section with an ID, so if i send you the link http://localhost:3000/#this-project, you will navigate to the section with the id <section id="this-project"> but as this is in a client component insde a <motion.div> i need to replicate the behavior manually i think

**Reply by sanalpanicker** [reply]
yes. I updated my comment.

https://nextjs.org/docs/pages/api-reference/components/link#:~:text=of%20the%20page-,The%20default%20behavior%20of,%3A,-%3CLink%20href

this might help?
